docs/field-values.asciidoc

```diff
@@ -341,6 +370,16 @@ The change event type is used for the subset of events within a category that in
 
 
 
+[float]
+[[ecs-event-type-connection]]
+==== connection
+
+Used primary with `event.category:network` this value is used for the subset of network traffic that includes sufficient information for the event to be included in flow or connection analysis. Events in this subcategory will contain at least source and destination IP addresses, source and destination TCP/UDP ports, and will usually contain counts of bytes and/or packets transferred. Events in this subcategory may contain unidirectional or bidirectional information, including summary information. Use this subcategory to visualize and analyze network connections. Flow analysis, including Netflow, IPFIX, and other flow-related events fit in this subcategory. Note that firewall events from many NGFW devices will also fit into this subcategory.  A common filter for flow/connection information would be `event.category:network AND event.type:connection AND event.type:end` (to view or analyze all completed network connections, ignoring mid-flow reports). You can further distinguish connection events using the ECS `event.action` field, populating with values of your choosing, such as "timeout", or "reset".
```

Comment:
`s/primary/primarily/`

Reply:
NGFW - if this is the first mention this also include Next-Generation Firewall.

Reply:
good, spelled out for first-time use.